spec/requests/requests_requests_spec.rb

```diff
@@ -41,6 +41,23 @@
           expect(response.body).to include('Print Unfulfilled Picklists (2)')
         end
       end
+
+      context "when there is a filter applied" do
+        let(:request) {
+          create(:request, partner_user: ::User.partner_users.first)
+          create(:request, partner_user: ::User.partner_users.last)
+        }
+
+        it "shows print unfulfilled picklists button with correct quantity when filtered" do
+          Request.delete_all
```

Comment:
Why do you need to do this? if there are requests created by previous parent blocks, maybe this should be outside of those blocks?